src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -411,8 +409,33 @@ def rm_dir(self, fs_id, path):
         cfs = self._cephfs_instance(fs_id)
         cfs.rm_dir(path)
 
-    @RESTController.Resource('POST')
-    def mk_snapshot(self, fs_id, path, name=None):
+    @RESTController.Resource('PUT', path='/quota')
+    def quota(self, fs_id, path, max_bytes=None, max_files=None):
+        """
+        Set the quotas of the specified path.
+        :param fs_id: The filesystem identifier.
+        :param path: The path of the directory/file.
+        :param max_bytes: The byte limit.
+        :param max_files: The file limit.
+        """
+        cfs = self._cephfs_instance(fs_id)
+        return cfs.set_quotas(path, max_bytes, max_files)
+
+    @RESTController.Resource('GET', path='/quota')
+    def get_quota(self, fs_id, path):
```

Comment:
You should remove `get_quotas` on line 470 than. (I bet it sneaked back in during the rebase)